src/main/java/com/coyoapp/tinytask/web/TaskController.java

```diff
@@ -44,4 +38,11 @@ public void deleteTask(@PathVariable String taskId) {
     log.debug("deleteTask(taskId={})", taskId);
     taskService.deleteTask(taskId);
   }
+
+  @ResponseStatus(HttpStatus.OK)
+  @PostMapping("/marktaskdone/{taskId}")
```

Comment:
There are a few things that come to mind here:
- as this is basically an update of a given task, PUT would be more appropriate instead of POST
- following API guidelines, the {taskId} should be at the start of the mapping
- naming could also be simplified, as the "task" is already contained in the mapping definition of the controller - one could go for `@PutMapping("/{taskId}/markdone")` or similar